modules/sync/protocolbuffers/wellknowntypes/state.json

```diff
@@ -399,6 +399,10 @@
     {
       "name": "v27.4",
       "digest": "e4017a7ca5cd7fc4fe75bd25b7c9ac5c321781e10fbddc954b1695cd0bccb36b30ebdfb0710a517c75a116f1d7ca67ca6ae20ff04305c257c515fafdb54020dc"
+    },
+    {
+      "name": "v28.0",
+      "digest": "e4bb315f5a90aace88fe39709c831eda8eb0ce66b4cf947065888100b9867c2eb9a0f61f6b8d73c51b1fcccdfe8611090f55ce5db1aee6901ec9b4e6d8fa8e52"
```

Comment:
I sanity checked with `v28.0` release [here](https://github.com/protocolbuffers/protobuf/releases/tag/v28.0) and looks like all files in `include` are present in our manifest. cc @pkwarren @jhump

Reply:
Double checked and things look good.